pkg/manifests/manifests.go

```diff
@@ -74,36 +74,56 @@ func Get(ns string, sharedCPUs string) (*Manifests, error) {
 		}
 	}
 
-	if ns != "" {
+	// by default, set the namespace to default namespace.
+	// the default namespace can be changed via Optional Parameter Function
+	WithNamespace("")(&mf)
```

Comment:
this is awkward to read though. I'd extract a common helper or even duplicate the namespace setting.